macros/edr/data_monitoring/monitors_query/table_monitoring_query.sql

```diff
@@ -85,6 +85,8 @@
             source_value,
             edr_bucket as bucket_start,
             {{ elementary.timeadd('day',1,'edr_bucket') }} as bucket_end,
+            {{ elementary.null_string() }} as dimension,
+            {{ elementary.null_string() }} as dimension_value,
```

Comment:
I prefer to be consistent, if at the final CTE these columns are last, they should be last here as well.
It will prevent future mistakes when we make changes.